src/include/distributed/connection_management.h

```diff
@@ -200,6 +200,9 @@ extern int NodeConnectionTimeout;
 /* maximum number of connections to cache per worker per session */
 extern int MaxCachedConnectionsPerWorker;
 
+/* maximum lifetime of connections in miliseconds */
```

Comment:
I think we could squeeze a test here: https://github.com/citusdata/citus/blob/master/src/test/regress/sql/shared_connection_stats.sql#L521

where we can set `max_cached_connection_lifetime` to 0, execute a query and  show that there are no connections after the query finishes?

(and also maybe use the default value and show that we have a cached connection)

Reply:
added